x/mint/keeper/genesis.go

```diff
@@ -18,6 +18,13 @@ func (k Keeper) InitGenesis(ctx sdk.Context, data *types.GenesisState) {
 	k.SetMinter(ctx, data.Minter)
 	k.SetParams(ctx, data.Params)
 
+	if err := k.SetTruncationDelta(ctx, types.DeveloperVestingModuleAcctName, sdk.ZeroDec()); err != nil {
+		panic(err)
+	}
+	if err := k.SetTruncationDelta(ctx, types.ModuleName, sdk.ZeroDec()); err != nil {
+		panic(err)
+	}
+
```

Comment:
Can we just add a comment above each of these saying that the top one is initializing developer module account and the next one is initializing the mint module account? It just threw me off initially because the top one is explicitly calling the module account while the second one is calling for the entire module name